README.md

```diff
@@ -246,12 +246,37 @@ Access to the s3 bucket is controlled by overriding the `awsCredentials` and `aw
 `AuthActions` sub trait in the play implementation).
 
 * **awsCredentials** defaults to None - this means that the instance profile of your app running in EC2 will be used. You can configure access to the bucket
-in your cloud formation script. For apps tha are not running in EC2 (such as developer environments) you can supply `BasicAWSCredentials` with a key and secret
+in your cloud formation script. For apps that are not running in EC2 (such as developer environments) you can supply `BasicAWSCredentials` with a key and secret
 for a user that will grant access to the bucket.
 
 * **awsRegion** defaults to eu-west-1 - This is where the guardian runs the majority of it's aws estate so is a useful default for us.
 
+### Cross-account access to S3 buckets
 
+Note that if your application is not running in the same AWS account as the S3 bucket then you will need to ensure two policy statements are in place. The first
+is a policy on the S3 bucket itself permitting access from a different AWS account. The second is the usual IAM policy defined in your cloudformation.
+
+You can add the first of these policies using the S3 console (in the account that owns the bucket). Inspect the bucket in the console and under _Permissions_
+select _Edit bucket policy_.
+
+Add something like the following statement to the bucket policy (if there is no existing policy then you'll need the skeleton in place as this fragment is only
+a statement).
+
+```
+{
+  "Sid": "gu-aws-composer-account",
+  "Effect": "Allow",
+  "Principal": {
+    "AWS": "arn:aws:iam::743583969668:root"
+  },
```

Comment:
+1 this approach is much nicer than passing aws keys around. 

Just wondering, does this give read access to all users of the account the private file?